lib/scrabble.rb

```diff
@@ -15,4 +13,19 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+
+  def score(word)
```

Comment:
Really like the clarity of thought that I see in this method, but would like it if each step was broken out into a smaller method. Could see methods that accomplish each of the following.

1) Check for nil and empty string.
2) Split word into letters.
3) Score letters.
4) Sum score values.